resources/views/components/actions.blade.php

```diff
@@ -7,7 +7,7 @@
 <div>
     @if(isset($actions) && count($actions) && $row !== '')
         @foreach($actions as $key => $action)
-            <td wire:key="action-{{ $key }}" class="pg-actions {{ $theme->table->tdBodyClass }}"
```

Comment:
I found the `wire:key` property sometimes made the action button to disappear and reappear on each button click. I removed the `wire:key` property and the button never went disappear anymore.
However, I am a bit skeptical with this solution as Livewire documentation itself already stated to include `wire:key` for each iteration (maybe for uniqueness?).
Read more here, https://laravel-livewire.com/docs/2.x/troubleshooting#dom-diffing-symptoms

If only there is a better workaround while keeping the codebase tied to convention then it would be delightful.

Reply:
For better visualization of the issue;

The first click on the first row of the table;
![Screenshot from 2022-02-28 16-56-17](https://user-images.githubusercontent.com/54124001/155953777-6d5e4513-4722-4b47-8c71-fc2b71873b9e.png)

The second click on the first row of the table;
![Screenshot from 2022-02-28 16-56-27](https://user-images.githubusercontent.com/54124001/155953887-71a1f89b-6cdb-4585-ae78-26567e804331.png)

Reply:
I believe this was giving the disappearing problem at times. In my case it didn't happen.